src/errors.ml

```diff
@@ -75,14 +75,35 @@ let print ppf loc =
           Format.fprintf pp "%s\n%*s\n" line
             stop_c
             (String.make len '^')
-        else if num_lines <= 10 then
-          let lines = file_lines path ~start:start.pos_lnum ~stop:stop.pos_lnum in
-          let last_lnum = Option.map ~f:fst (List.last lines) in
-          let padding_width = Option.value_exn
-                                (Option.map ~f:String.length last_lnum) in
-          List.iter ~f:(fun (lnum, l) ->
-            Format.fprintf pp "%*s: %s\n" padding_width lnum l)
-          lines
+        else
+          let get_padding lines =
+            let last_lnum = Option.map ~f:fst (List.last lines) in
+            Option.value_exn (Option.map ~f:String.length last_lnum)
```

Comment:
If you call `value_exn` on the first option, you'll be able to remove the second `Option.map`.